File: events-log-service/src/main/java/com/nurdor_project/events_log_service/dto/EventDto.java
package com.nurdor_project.events_log_service.dto;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class EventDto {
    @JsonProperty("id")
    int id;
    @JsonProperty("eventName")
    String eventName;
    @JsonProperty("description")
    String description;
    @JsonProperty("isStarted")
    Byte isStarted;
    @JsonProperty("startTime")
    String startTime;
    @JsonProperty("endTime")
    String endTime;
    @JsonProperty("latitude")
    double latitude;
    @JsonProperty("longitude")
    double longitude;
    @JsonIgnore
    @JsonProperty("eventImg")
    String eventImg;
    @JsonProperty("locationDesc")
    String locationDesc;
    @JsonProperty("totalDonations")
    Long totalDonations;
    @JsonProperty("city")
    String city;
}

File: mobile-app/app/src/main/java/com/example/nurdor_volunteer_app_v3/dto/EventDto.kt
package com.example.nurdor_volunteer_app_v3.dto

data class EventDto(
    val id: Int,
    val eventName: String,
    val description: String,
    val startTime: String,
    val endTime: String,
    val latitude: Double,
    val longitude: Double,
    val eventImg: String?,
    val locationDesc: String?,
    val city: String)
